Gemfile

```diff
@@ -10,11 +10,15 @@ group :development do
   gem 'mutant-rspec',  '~> 0.8.8'
   gem 'rake',          '~> 11.1'
   gem 'rspec',         '~> 3.0'
-  gem 'rubocop',       '~> 0.42.0'
   gem 'simplecov',     '~> 0.12.0'
   gem 'yard',          '~> 0.9.5'
   gem 'activesupport', '~> 4.2'
 
+  if RUBY_VERSION >= '2.2'
```

Comment:
Do you mean where it says 'Install rubocop to run its rake tasks'? That is output by the dummy rake task I generate to avoid having to fiddle with the `:ci` task. I'm not too happy with how that turned out, suggestions are welcome. One option is to add the `:rubocop` task to the `:ci` task when it is defined, but that doesn't make it easier to see what `rake ci` will do.